src/Compilers/CSharp/Portable/Syntax/InternalSyntax/CSharpSyntaxNode.cs

```diff
@@ -175,7 +175,7 @@ internal static DirectiveStack ApplyDirectivesToListOrNode(GreenNode listOrNode,
 
         internal virtual IList<DirectiveTriviaSyntax> GetDirectives()
         {
-            if ((this.flags & NodeFlags.ContainsDirectives) != 0)
+            if (this.ContainsDirectives)
```

Comment:
a tiny handful of cases were examining the flags, instead of the existing api that more easily and correctly checks these cases.